aptos-move/block-executor/src/view.rs

```diff
@@ -523,14 +504,22 @@ impl<'a, T: Transaction, X: Executable> ParallelState<'a, T, X> {
             }
         }
     }
+}
+
+impl<'a, T: Transaction, X: Executable> ResourceState<T> for ParallelState<'a, T, X> {
+    fn set_base_value(&self, key: T::Key, value: ValueWithLayout<T::Value>) {
+        self.versioned_map.data().set_base_value(key, value);
+    }
 
     /// Captures a read from the VM execution, but not unresolved deltas, as in this case it is the
```

Comment:
Comment seems outdated, e.g. the caller is calling read_cached_data_by_kind, not fetch_data.
Also it's not just about aggregators, but all base values, seemingly.